index.d.ts

```diff
@@ -375,6 +375,10 @@ declare module 'plaid' {
     warnings: Array<Warning>;
   }
 
+  interface AssetReportGetPdfResponse extends BaseResponse {
```

Comment:
thanks for adding these typings 👍 we should definitely add them regardless of what we land on w.r.t. the buffer.